lab_3_ann_retriever/main.py

```diff
@@ -104,6 +107,15 @@ def tokenize(self, text: str) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(text, str) or not text:
+            return None
+        text = text.lower()
+        tokenized_text = ''
```

Comment:
do you really need to create separate variable `tokenized_text`?

Reply:
I have 2 options. Second one is shorter, but u said that it is too difficult last time